lib/kubeclient/common.rb

```diff
@@ -204,6 +204,16 @@ def retrieve_all_entities(entity_types)
         end
       end
 
+      def proxy_url(kind, name, port, namespace = '')
```

Comment:
I think that it's confusing to call it "kind" and expect something pluralized from the user